README.md

```diff
@@ -346,31 +346,31 @@ npm start -- [options]
+| <pre nowrap>--peers<br> -x</pre>     | LISK_PEERS             | peers.list               | Comma separated list of peers to connect to in the format `192.168.99.100:5000,172.169.99.77:5000`                                                                                                                                                                                                                                               |
+|                                      | LISK_API_PUBLIC        | api.access.public        | Enable or disable public access of http API. Must be set to true/false                                                                                                                                                                                                                                                                           |
+|                                      | LISK_API_WHITELIST     | api.access.whiteList     | Comma separated list of IPs to enable API access. Format `192.168.99.100,172.169.99.77`                                                                                                                                                                                                                                                          |
+|                                      | LISK_FORGING_DELEGATES | forging.delegates        | Comma separated list of delegates to load in the format _publicKey&#x7c;encryptedPassphrase,publicKey2&#x7c;encryptedPassphrase2_                                                                                                                                                                                                                |
+|                                      | LISK_FORGING_WHITELIST | forging.access.whiteList | Comma separated list of IPs to enable access to forging endpoints. Format `192.168.99.100,172.169.99.77`                                                                                                                                                                                                                                         |
+| <pre nowrap>--rebuild</pre>          |                        |                          | Rebuild the chain from Genesis block up to specified round number, must be a positive integer equal to or greater than `0`. When `0` is passed, this corresponds to the inclusion of all rounds. Any other number equals to its corresponding round. Bear in mind this mode disables all the network features of the node to ensure reliability. |
```

Comment:
I'm not so sure about the wording here. I'd suggest `--validate -v`. What we do it's to process again and therefore validate every block until the typed round.

Reply:
So, rename the functions as well: `createSnapshot` -> `rebuildChain`.
Your comment about documentation wording is correct. We both validate and rebuild up to the specified number.